neo/Network/RPC/RpcClient.cs

```diff
@@ -252,39 +252,39 @@ public RpcInvokeResult InvokeFunction(string address, string function, RpcStack[
         /// Returns the result after passing a script through the VM.
         /// This RPC call does not affect the blockchain in any way.
         /// </summary>
-        public RpcInvokeResult InvokeScript(string script)
+        public virtual RpcInvokeResult InvokeScript(byte[] script)
         {
-            return RpcInvokeResult.FromJson(RpcSend("invokescript", script));
+            return RpcInvokeResult.FromJson(RpcSend("invokescript", script.ToHexString()));
         }
 
         /// <summary>
         /// Returns a list of plugins loaded by the node.
         /// </summary>
-        public RpcPlugin[] ListPlugins()
+        public virtual RpcPlugin[] ListPlugins()
         {
             return ((JArray)RpcSend("listplugins")).Select(p => RpcPlugin.FromJson(p)).ToArray();
         }
 
         /// <summary>
         /// Broadcasts a transaction over the NEO network.
         /// </summary>
-        public bool SendRawTransaction(string rawTransaction)
+        public virtual bool SendRawTransaction(string rawTransaction)
         {
             return RpcSend("sendrawtransaction", rawTransaction).AsBoolean();
         }
 
         /// <summary>
         /// Broadcasts a raw block over the NEO network.
         /// </summary>
-        public bool SubmitBlock(string block)
+        public virtual bool SubmitBlock(string block)
         {
             return RpcSend("submitblock", block).AsBoolean();
         }
 
         /// <summary>
         /// Verifies that the address is a correct NEO address.
         /// </summary>
-        public RpcValidateAddressResult ValidateAddress(string address)
+        public virtual RpcValidateAddressResult ValidateAddress(string address)
```

Comment:
All these virtual are being reimplemented?

Reply:
I don't think so

Reply:
The virtual method is just for unit test Moq😂

Reply:
I don't like these `virtual`s.

Reply:
Removed ```virtual```s from the methods, but changed ```RpcSend``` to be virtual for UT.